modules/ROOT/pages/secure-configuration-properties.adoc

```diff
@@ -3,7 +3,11 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-Secure properties can encrypt properties and store the encrypted text in secure properties files.
+Create secure configuration properties by encrypting them. To set up the encryption process:
+
+. Obtain the Mule Secure Configuration Property Extension module from Exchange.
```

Comment:
Seems like having that link to <<Adding the Secure Configuration Properties Module>> (in step 2) here instead might head off questions about how to do that.

Reply:
Also, to make a less fragile link, I'd recommend putting an anchor above the section and linking to it that way, e.g., <<link_to_anchor>>.

Reply:
Changed